### DISCUSSION TITLE
The "shorten URL" button is not responding

### DISCUSSION THREAD

**Discussion by dutoitlilo** [community/community#37327] [Other Feature Feedback, Questions, & Ideas]
I'm at the end of a document where I used GitHub to shorten URLs many times without any issues at all. But today, all the info stays the same on the form, the URL is just different, and the "shorten URL" button is just not responding at all. I've tried taking out all the info in the boxes not marked with an asterisk and tried different URLs that I used yesterday, and which worked fine, but today it doesn't wanna...is there something up with the button itself? I've tried in both Chrome and Edge. 
![campaign builder werk nie en ek is op die laaste snippet](https://user-images.githubusercontent.com/116723546/198009579-a6d2ae00-5308-40f6-b9b4-b412e33c4d76.jpg)
![campaign builder wil nie werk vanoggend nie 3](https://user-images.githubusercontent.com/116723546/198009589-8bb90d7d-7585-4c77-af2a-6f333216d310.jpg)
![campaign builder wil nie werk vanoggend nie 4](https://user-images.githubusercontent.com/116723546/198009591-fd18717e-969b-44b5-8375-54bd2a9e53cb.jpg)
![campaign builder wil nie werk vanoggend nie](https://user-images.githubusercontent.com/116723546/198009603-8d951c1b-3737-47c5-92d6-3b2f601356ca.jpg)
![campaign manager shorten link button reageer gladnie 1](https://user-images.githubusercontent.com/116723546/198009605-796d6c18-38ce-4f73-b2c3-54fa9a2ecb39.jpg)
![campaign manager shorten link button reageer gladnie 2](https://user-images.githubusercontent.com/116723546/198009611-d5fbd71c-3dc4-4fa2-bec8-605c1833114e.jpg)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @dutoitlilo ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
